driver/csiplugin/controllerserver.go

```diff
@@ -721,7 +721,11 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		policy.Partition = fmt.Sprintf("csi-T%s", scaleVol.Tier)
 
 		scaleVol.VolName = fmt.Sprintf("%s-T%scsi", scaleVol.VolName, scaleVol.Tier)
-		scaleVol.Connector.SetFilesystemPolicy(&policy, scaleVol.VolBackendFs)
+		err = scaleVol.Connector.SetFilesystemPolicy(&policy, scaleVol.VolBackendFs)
```

Comment:
The handling of the SetFilesystemPolicy err was also handled in the open PR#595
But these messages are better, so I can rebase that PR once this is merged.